.pre-commit-config.yaml

```diff
@@ -46,7 +46,7 @@ repos:
   hooks:
     - id: black
 - repo: https://github.com/PyCQA/pylint
-  rev: v2.13.8
+  rev: v2.14.0-b1
```

Comment:
Bot wants to upgrade to a beta version??


Looks like they forgot to tick the "pre-release" checkbox: https://github.com/PyCQA/pylint/releases/tag/v2.14.0-b1

Reply:
I think it's probably fine but we can also just skip this.